setup.py

```diff
@@ -239,6 +239,7 @@ def run(self):
 extras_require['all'] = everything
 
 if 'setuptools' in sys.modules:
+    setuptools_extra_args['python_requires'] = '>=3.3'
```

Comment:
This requires setuptools >= 24.2.0:
https://setuptools.readthedocs.io/en/latest/history.html#v24-2-0